stdlib/cache/src/main/ballerina/cache/cache.bal

```diff
@@ -211,8 +211,18 @@ function runCacheExpiry() returns error? {
     string[] emptyCacheKeys = [];
 
     // Iterate through all caches.
-    foreach var (currentCacheKey, currentCache) in cacheMap {
-
+    int keyIndex = 0;
+    string[] currentCacheKeys = cacheMap.keys();
+    while (keyIndex < currentCacheKeys.length()) {
+
+        Cache currentCache;
+        string currentCacheKey = currentCacheKeys[keyIndex];
+        Cache? cache = cacheMap[currentCacheKey];
+        if (cache is Cache) {
+            currentCache = cache;
+        } else {
+            return ();
```

Comment:
Why is there a return here? Shouldn't this be `continue`?

Reply:
Yes. Resolved